### PR TITLE
[Gameboard] Initializing empty gameboard

### PR DESCRIPTION
**Pull request by pranathip**
Added via GiphyTesting this pr description editor

### CODE REVIEW

File: src/GameBoard.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  
+  const getRandomPosition = () => {
+    const rowPosition = Math.floor(Math.random() * 4);
+    const colPosition = Math.floor(Math.random() * 4);
+    return [rowPosition, colPosition];
+  };
+  
+  export const generateRandom = (board) => {
+    if (isFull(board)) {
+      return board;
```
Comment (pranathip) [inline src/GameBoard.js:31]: test

File: src/GameBoard.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
+export const getEmptyBoard = () => [
+    [0, 0, 0, 0],
+    [0, 0, 0, 0],
+    [0, 0, 0, 0],
+    [0, 0, 0, 0]
+  ];
+  
+  const hasValue = (board, value) => {
```
Comment (pranathip) [inline src/GameBoard.js:8]: leaving a comment

File: src/GameBoard.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+      for (let j = 0; j < board[i].length; j++) {
+        if (board[i][j] === value) {
+          return true;
+        }
+      }
+    }
+    return false;
+  };
+  
+  export const isFull = (board) => {
```
Comment (pranasaurus-rex) [inline src/GameBoard.js:19]: this function looks good

File: src/GameBoard.js (hunk 1 above)
Comment (pranasaurus-rex) [reply src/GameBoard.js:31]: nice test!

**Review by pranasaurus-rex — APPROVED** [review]
LGTM!

**Comment by pranathip** [comment]
@pranathip started a stack merge that includes this pull request via [Graphite](https://app.stg.graphite.dev/github/pr/pranathip/pranathi-2048-react-app/5).

**Comment by pranathip** [comment]
@pranathip merged this pull request with [Graphite](https://app.stg.graphite.dev/github/pr/pranathip/pranathi-2048-react-app/5).

**Pull request merged**